compiler/luci/pass/include/luci/Pass/PropagateQuantParamPass.h

```diff
@@ -27,9 +27,17 @@ namespace luci
  */
 struct PropagateQuantParamPass final : public logo::Pass
 {
+  PropagateQuantParamPass(bool TF_style_maxpool) : _TF_style_maxpool(TF_style_maxpool) {}
+
+  // For backward compatibility
```

Comment:
This comment seems for the reviewer, about the change.
If you land this and look at this comment, it looks strange to me.
If you have other thoughts, could you please share it with me?

Reply:
I understand. I'll remove the comment.